File: langgraph/result/qwen/32b-eq2js/4.js
function calculateMeanAbsDev(values) {
    const average = values.reduce((sum, value) => sum + value, 0) / values.length;
    const deviationSum = values.reduce((sum, value) => sum + Math.abs(value - average), 0);
    return deviationSum / values.length;
}

File: langgraph/outputs/equiv/4.py
from typing import List

def compute_mean_abs_deviation(data_points: List[float]) -> float:
    average = sum(data_points) / len(data_points)
    total_dev = 0.0
    for value in data_points:
        total_dev += abs(value - average)
    return total_dev / len(data_points)
# method name: mean_absolute_deviation -> compute_mean_abs_deviation
# parameter name: numbers -> data_points
# variable name: mean -> average, sum(...) -> total_dev
# replaced generator expression with explicit for loop
# changed variable names and structure